aquarius/events/events_monitor.py

```diff
@@ -221,7 +226,26 @@ def process_block_range(self, from_block, to_block):
                     f"Error processing update metadata event: {e}\n" f"event={event}"
                 )
 
+        for event in self.get_event_logs(EVENT_METADATA_STATE, from_block, to_block):
```

Comment:
as previously mentioned, please remove some of the duplication for events processing, since all these fors are badsically the same thing with a different event name and processor class.